packages/appstore/src/components/cfds-listing/index.tsx

```diff
@@ -140,14 +140,24 @@ const CFDsListing = () => {
                         const has_mt5_account_status = existing_account.status
                             ? getMT5AccountAuthStatus(existing_account.status)
                             : null;
+                        const is_demo_account = existing_account.account_type === 'demo';
+                        const is_demo_get_account = is_demo && existing_account.action_type === 'get';
                         return (
                             <TradingAppCard
                                 action_type={existing_account.action_type}
                                 availability={selected_region}
                                 clickable_icon
                                 icon={existing_account.icon}
-                                sub_title={existing_account?.sub_title}
-                                name={!has_mt5_account_status ? existing_account?.name : ''}
+                                sub_title={
+                                    is_demo_account
+                                        ? `${existing_account?.sub_title} ${localize('Demo')}`
+                                        : existing_account?.sub_title
+                                }
+                                name={
+                                    !has_mt5_account_status && is_demo_get_account
+                                        ? `${existing_account?.name} ${localize(' Demo')}`
+                                        : existing_account?.name
+                                }
```

Comment:
Previously we displayed the name when mt5 account status was NULL. so wont this condition effect current prod behaviour. 



```suggestion
                        !has_mt5_account_status 
                                        ? {existing_account?.name} 
                                        : is_demo_get_account ?`${existing_account?.name} ${localize(' Demo')}:''`
```

Reply:
ok